source/usage-examples/findOne.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri = "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority"
+
+var ctx = context.TODO()
+
+func main() {
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI(uri))
+	if err != nil {
+		panic(err)
+	}
+
+	defer func() {
+		if err = client.Disconnect(ctx); err != nil {
+			panic(err)
+		}
+
```

Comment:
Sorry again for not clarifying before. By "logical block" I meant logical grouping of code to help readability, not a newline strictly after every if/else. I think this newline is extraneous.

Reply:
Comment: https://github.com/mongodb/docs-golang/pull/2#discussion_r622202935
Fix: https://github.com/mongodb/docs-golang/pull/2/commits/72cf9221c25b39c390a7d7162a0c64c03f75c549